bpython/curtsiesfrontend/repl.py

```diff
@@ -892,7 +958,15 @@ def only_whitespace_left_of_cursor():
             for unused in range(to_add):
                 self.add_normal_character(" ")
             return
-
+        # if cursor on closing character from pair,
+        # moves cursor behind it on tab
+        # ? should we leave it here as default?
```

Comment:
This feels good to me, is this question whether this behavior should be used even when brackets_completion is off? It seems like it'd be ok to keep while off to me.

Reply:
okay